.github/workflows/test_build_conda_m1.yml

```diff
@@ -40,7 +41,7 @@ jobs:
     with:
       conda-package-directory: ${{ matrix.conda-package-directory }}
       repository: ${{ matrix.repository }}
-      ref: nightly
+      ref: ${{ matrix.ref }}
```

Comment:
let's revert this back to nightly. (and all such changes in the other test workflow)